web/src/compose.js

```diff
@@ -373,6 +372,12 @@ function schedule_message_to_custom_date() {
     } else {
         req_type = message_type;
     }
+    //valueof() gives us the timestamp in milliseconds so dividing by 1000 gives us seconds
+    const now = new Date().valueOf() / 1000;
+    const minimum_schedule_msg_delay = scheduled_messages.MINIMUM_SCHEDULED_MESSAGE_DELAY_SECONDS;
+    const time_difference = Math.abs(scheduled_delivery_timestamp - now);
+    const add_notice = time_difference < minimum_schedule_msg_delay ? true : false;
```

Comment:
* `scheduled_delivery_timestamp` will always be greater than `now`, right? Does there exists a way via UI where the reverse is possible?
* > const minimum_schedule_msg_delay = scheduled_messages.MINIMUM_SCHEDULED_MESSAGE_DELAY_SECONDS;

I'm not sure if there is any real benefit of storing a constant value in another constant as we would be using it at 1 place only -- but no strong opinion. In any case, we should use `scheduled` and we avoid abbreviation like `msg` or `mssg`. Use `message`.
* I think we should use `scheduled_message_delay_seconds` as the variable named instead of `time_difference`. Then the code will be more readable as:
```js
const add_notice = scheduled_message_delay_seconds < scheduled_messages.MINIMUM_SCHEDULED_MESSAGE_DELAY_SECONDS ? true : false;
```
* `add_notice` -> `minimun_scheduled_message_delay_minutes_note`; because we show the banner everytime, we just include this note when such a condition is met